client/packages/system/src/Item/DetailView/Tabs/ItemVariants/ItemVariantsTab.tsx

```diff
@@ -102,9 +102,8 @@ const ItemVariant = ({
             label={t('label.temperature')}
             labelWidth="200"
             Input={
-              // TODO: temp range dropdown
               <BasicTextInput
-                value={variant.coldStorageTypeId}
+                value={variant.coldStorageType?.name ?? ''}
```

Comment:
Do you think it's worth showing the temperature here too? e.g. "Name (-10°C – 0°C)"

Or is the name supposed to have that info in it?

Reply:
I think for GAPS the idea was just to use name, but I agree would be nice to show in this view. Will update.

Reply:
Actually on second thoughts, I don't think it's important in this view. The names should be descriptive (hopefully!)